include/zephyr/dt-bindings/usb/video.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024 tinyVision.ai Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_USB_VIDEO_H
+#define ZEPHYR_INCLUDE_DT_BINDINGS_USB_VIDEO_H
+
+/* UVC GUIDs as defined on Linux source */
```

Comment:
What is "Linux source"? Linux Kernel? Version? Any license issue?

Reply:
The idea was to preserve GUIDs compatible with Linux: is reproducing a list of IDs to be the same as GPL code causing issue? I am glad to find other sources. Maybe growing this list over time as contributions come is better?

We may also be able to skip most of this list by generating the GUIDs: they are often the four letter code of the video format, followed by some byte sequence from the UVC standard.

![scrot_20241011_102821_612x174](https://github.com/user-attachments/assets/2e8017fb-f523-4b82-882c-e556282b3110)

```
$ printf '\x32\x59\x55\x59\x00\x00\x00\x10\x80\x00\x00\xAA\x00\x38\x9B\x71' | hexdump -C
00000000  32 59 55 59 00 00 00 10  80 00 00 aa 00 38 9b 71  |2YUY.........8.q|
                                                                ^
                                                           Format code in little endian
```

Reply:
The new approach is to always use the [Media Foundation GUID format](https://github.com/gix/media-types), and decide later what to do when a non-canonical format needs to be used. This permit to skip this large table: nothing to import from anywhere.